docs/setup_wsl.md

```diff
@@ -81,11 +82,16 @@ The first time you open Ubuntu, you will be prompted to create a Linux username
 Double check that your user has been created.
 ```console
 $ whoami
-awdeorio  // this should NOT say root
+awdeorio  # this should NOT say root
 ```
 
 <div class="primer-spec-callout warning" markdown="1">
 **Pitfall:** If you are signed in as [root](#root-user), something has gone wrong. We recommend that you [uninstall](https://www.minitool.com/news/uninstall-wsl.html) and [reinstall](#install-wsl) WSL to try again.
+```console
+$ whoami
+root  # SOMEthiNG IS WRONG
```

Comment:
Is the mixed casing in "SOMEthiNG IS WRONG" intentional?

Reply:
Yes, but please go ahead and change it to your preference.  No big deal.

Reply:
Nice, lol